stringResources/en-US/resources.resjson

```diff
@@ -6,4 +6,9 @@
   "Visual_Y": "Y",
   "Visual_ForceDisplay": "Force display",
   "Visual_EnableColorbrewer": "Enable colorbrewer"
+	"Visual_General": "General",

```

Comment:
There's the same key at the second line.
Please double check that there're no duplicates. Otherwise, the JSON file will be invalid